target/rv32imc/riscvOVPsim.ic

```diff
@@ -0,0 +1,22 @@
+# riscOVPsim configuration file converted from YAML
+--variant RVB32I
```

Comment:
Does this include RV32 M,C extension?

Reply:
the target/rv32imc/riscvOVPsim.ic is derived from the riscv-config yaml specs that defines all legal options that a device can have and still meet the riscv specs.

in the riscvOVPsim.ic files there are several important statements that define the extensions. these are all documented in the riscvOVPsim docs, and also if you use , eg
   cd riscv-ovpsim/bin/Linux64
    riscvOVPsim.exe --showoverrides.

--variant RVB32I   # says this is a base rv32i - ie no U no S etc...  - the simplest.
--override riscvOVPsim/cpu/add_Extensions=MC # says add on M and C extensions
--override riscvOVPsim/cpu/misa_MXL=1 # and just confirm 32bits

so yes, to answer your question, Does this include RV32 M,C extension? it does.

thanks
Simon